components/vf-tree/vf-tree.scss

```diff
@@ -63,6 +63,11 @@
     transform: rotate(90deg);
     }
   }
+  &:focus:focus-visible {
```

Comment:
I would wrote this one as: 
`&:focus:focus-visible svg {
    outline: -webkit-focus-ring-color auto 1px;
  }`
since it only has svg rule as its only child. It can stay as it is.